apps/rig_inbound_gateway/config/config.exs

```diff
@@ -94,7 +94,8 @@ config :rig, RigInboundGateway.ApiProxy.Router,
 config :rig, RigInboundGateway.ApiProxy.Handler.Http,
   cors: {:system, "CORS", "*"},
   kafka_response_timeout: {:system, :integer, "PROXY_KAFKA_RESPONSE_TIMEOUT", 5_000},
-  kinesis_response_timeout: {:system, :integer, "PROXY_KINESIS_RESPONSE_TIMEOUT", 5_000}
+  kinesis_response_timeout: {:system, :integer, "PROXY_KINESIS_RESPONSE_TIMEOUT", 5_000},
+  http_response_timeout: {:system, :integer, "PROXY_HTTP_RESPONSE_TIMEOUT", 5_000}
```

Comment:
if you decide to go with `http_async`, this could be called `http_async_response_timeout` to differentiate it from the timeout used when waiting for the _synchronous_ response